tests/suit/migration_test.go

```diff
@@ -57,5 +57,12 @@ var _ = Describe("[level:component]Migration tests for vSphere provider", func()
 		err = utils.WaitForPlanReadyWithTimeout(f.CrClient, f.Namespace.Name, "plan-test", 15*time.Second)
 		Expect(err).ToNot(HaveOccurred())
 
+		By("Creating migration")
+		migrationDef := utils.NewMigration(provider.Namespace, "migration-test", "plan-test")
```

Comment:
I see the test keeps failing but unrelated to that - please define a constant instead of duplicating "plan-test" as suggested by SonarCloud

Reply:
moved the managed objects names to constants, 
i think its failing because  of the conversion v2v Pod  still requests  kvm caps.

Reply:
it seems we were missing this part https://github.com/kubev2v/forklift/pull/106